README.md

```diff
@@ -47,6 +47,15 @@ This provider's versions are able to install and manage the following versions o
 | Kubernetes 1.21 | ✓ | ✓ | ✓ |
 | Kubernetes 1.22 |   | ✓ | ✓ |
 
+<aside class="note warning">
+
+<h1> Warning </h1>
+
+Managed Clusters (AKS) support only some patch versions from the above table. For example, while an unmanaged cluster can use every 1.21 patch version, from 1.21.0 to 1.21.9, an AKS cluster may use only patch versions 1.21.2 or 1.21.7.
```

Comment:
maybe we should stay clear of naming specific versions and instead link to https://docs.microsoft.com/en-us/azure/aks/supported-kubernetes-versions?tabs=azure-cli#kubernetes-version-support-policy with a generic statement about how AKS follows its own support policy? those specific example versions will quickly become unsupported and it might confuse users

Reply:
Fixed